build/package/.vscodeignore

```diff
@@ -17,8 +17,8 @@
 */**
 !dist/ext/extension.js
 !images/*
-!LICENSE
-!NOTICE
+# !LICENSE
+# !NOTICE
```

Comment:
Yeah I added the ignore to make sure the build/package ones were being copied in, I will remove these comments but I think the package script will need to ignore them as it will copy in the root directory ones

Reply:
As long as `copyGlob("*", "build/package")` is after the vscodeignore copy it should be fine--it should just overwrite the ones copied from the root. Which is maybe a nice feature and an easy way to override files copied from the root if needed (e.g. maybe we'll want to to do that for the README file as well (see issue #100)